src/QMCHamiltonians/QMCHamiltonian.cpp

```diff
@@ -1095,7 +1095,7 @@ void QMCHamiltonian::evaluate(MCWalkerConfiguration& W,
     ScopedTimer h_timer(my_timers_[i]);
     H[i]->addEnergy(W, LocalEnergyVector, Txy);
   }
-  KineticEnergyVector = H[0]->ValueVector;
```

Comment:
@PDoakORNL can you confirm if removing `OperatorBase::ValueVector` is reasonable? GPU CI is passing now and it's commented out in line 1057.

Reply:
Do it as separate commit please